website/docs/plugins/ray_launcher.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
This readme is becoming big enough to possibly split into multiple files.
no need to do it yet, but think about it.